src/components/verification-grid/verification-grid.ts

```diff
@@ -2275,29 +2378,60 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
     return html`<div class="grid-tile tile-placeholder">${this.emptySubjectText}</div>`;
   }
 
+  private defaultGridTileTemplate(): HTMLTemplateResult {
```

Comment:
this is neater. Might make this public later if want to say emit a default template for a configurator tool or something

Reply:
could this be static?

Reply:
> could this be static?

Yes, there's even static templates in Lit (https://lit.dev/docs/api/static-html/), this might be a perfect use case

Reply:
cool!